src/class/object_individuals.php

```diff
@@ -197,6 +199,27 @@ function __construct ()
                       ));
         $this->sSortDefault = 'id';
 
+        // For installations with Phenotype/Additional enabled, link to that column as well and show it,
+        //  so users can quickly identify individuals with certain features.
+        // Simplest is to check for ourselves, so we don't need to initiate any object.
+        // FIXME: Should this be replaced by a CustomVL, with Ind joined to Phenotypes to create this?
+        // FIXME: This Ind VL has more than that Ind VL, though.
+        if ($_DB->query('SELECT COUNT(*) FROM ' . TABLE_ACTIVE_COLS . ' WHERE colid = ?', array('Phenotype/Additional'))->fetchColumn()) {
+            // Column is active, include in SELECT, JOIN and the column list.
+            $this->aSQLViewList['SELECT'] .= ', GROUP_CONCAT(DISTINCT p.`Phenotype/Additional` ORDER BY p.`Phenotype/Additional` SEPARATOR ", ") AS phenotypes_';
+
+            $this->aSQLViewList['FROM'] .= ' LEFT OUTER JOIN ' . TABLE_PHENOTYPES . ' AS p ON (i.id = p.individualid';
+            if (true || $_AUTH['level'] < $_SETT['user_level_settings']['see_nonpublic_data']) { // This check assumes lovd_isAuthorized() has already been called for gene-specific overviews.
```

Comment:
I presume `true` here is a remnant of some debugging code.

Other than that, I think it's good to merge.

Reply:
Haha whoops, thanks for picking that up :sweat_smile: